src/threading.ts

```diff
@@ -0,0 +1,110 @@
+import interfaces = require('./interfaces');
+import IPy_Object = interfaces.IPy_Object;
+import Py_CodeObject = require('./codeobject');
+import Py_FuncObject = require('./funcobject');
+import opcodes = require('./opcodes');
+import optable = require('./optable');
+import Py_Cell = require('./cell');
+import Py_FrameObject = require('./frameobject');
+import enums = require('./enums');
+
+var maxMethodResumes: number = 10000,
+// The number of method resumes until Doppio should yield again.
+methodResumesLeft: number = maxMethodResumes,
```

Comment:
Nit: These should be tabbed in by one to show that they are part of the `var` declaration.